packages/wrangler/src/r2/helpers.ts

```diff
@@ -299,3 +299,14 @@ export async function putR2Sippy(
 		{ method: "PUT", body: JSON.stringify(config), headers }
 	);
 }
+
+/**
+ * R2 bucket names must only contain alphanumeric and - characters.
+ */
+export function isValidR2BucketName(
+	name: string | undefined
+): name is string {
+	return (
+		typeof name === "string" && /^[a-zA-Z][a-zA-Z0-9-]*$/.test(name)
```

Comment:
```suggestion
		typeof name === "string" && /^[a-zA-Z][a-zA-Z0-9-]+$/.test(name)
```

Reply:
Can a bucket not have a single character for its name?